tcs/csp_solver_two_tank_tes.cpp

```diff
@@ -120,129 +241,118 @@ void C_heat_exchanger::hx_performance(bool is_hot_side_mdot, bool is_storage_sid
 	// Outputs: HX effectiveness [-], hot side outlet temp [K], cold side outlet temp [K], 
 	//				Heat transfer between fluids [MWt], cold side mass flow rate [kg/s]
 
-    if (m_dot_known < 0) {
-        eff = T_hot_out = T_cold_out = q_trans = m_dot_solved = std::numeric_limits<double>::quiet_NaN();
-        throw(C_csp_exception("HX provided a negative mass flow", ""));
-    }
-    else if (m_dot_known == 0) {
-        eff = 0.;
-        T_hot_out = T_hot_in;
-        T_cold_out = T_cold_in;
-        q_trans = 0.;
-        m_dot_solved = 0.;
-        return;
-    }
+	if (m_dot_known < 0) {
+		eff = T_hot_out = T_cold_out = q_trans = m_dot_solved = std::numeric_limits<double>::quiet_NaN();
+		throw(C_csp_exception("HX provided a negative mass flow", ""));
+	}
+	else if (m_dot_known == 0) {
+		eff = 0.;
+		T_hot_out = T_hot_in;
+		T_cold_out = T_cold_in;
+		q_trans = 0.;
+		m_dot_solved = 0.;
+		return;
+	}
```

Comment:
Fix your tab to spaces setting.

Reply:
VS not cooperating

Reply:
Mine neither. I'm sure converting tabs to spaces will be a perpetual (but easy) task.